mozilla/foundation.mozilla.org
# address all acceptance criteria

Closes https://github.com/mozilla/foundation.mozilla.org/issues/6928

Testing:
- check out `wagtail-localize-2021` branch
- clone the current `foundation-localize` app's database
- docker compose up
- load http://localhost:8000/cms/pages/search/?content_type=wagtailpages.softwareproductpage
- open enough of these to get a mix of "yes", "no", and "unknown" for medical compliance
- docker compose down while leaving your tabs open,
- check out this branch
- apply migrations 
- docker compose up
- reload your tabs: the values still still be yes/no/unknown
- load the CSM edit pages for each product
- the pickers should still be dropdown selects with options "Yes", "No", and "Unknown".
